odoo/addons/base/models/ir_ui_view.py

```diff
@@ -595,6 +595,12 @@ def write(self, vals):
         if vals.get('active') or 'arch_db' in vals:
             self.filtered('active')._check_xml()
 
+        for view in self:
+            if view.type in ('form', 'list') and view.model_data_id and view.model_data_id.module == 'studio_customization' and view.invalid_locators:
+                filtered = list(filter(lambda loc: loc.get('attrib', {}).get('position') != 'move', view.invalid_locators))
+                if filtered:
+                    raise ValidationError(_("Invalid XPath in Studio view '%(view)s' (line %(line)s).") % {'view': view.name, 'line': filtered[0].get('sourceline')})
+
```

Comment:
Instead you should raise error during write/save view (or may be constraint) by checking `invalid_locators` exists or not.
https://github.com/odoo/odoo/commit/040aec546d29a511d7f117b676796bd2b1cad285